src/ctl/src/cmd_impl/hummock/sst_dump.rs

```diff
@@ -39,10 +45,25 @@ pub async fn sst_dump() -> anyhow::Result<()> {
             let sstable_cache = sstable_store
                 .sstable(id, &mut StoreLocalStatistic::default())
                 .await?;
-            let sstable_meta = &sstable_cache.value().meta;
+            let sstable = sstable_cache.value().as_ref();
+            let sstable_meta = &sstable.meta;
+
+            let sstable_id_info = sstable_id_infos_iter.next().unwrap();
```

Comment:
Does `sstable_id_infos` have the same length as SUM of `length of level.table_infos` on all levels?

If so, we may flat the two loops into one and use `zip_eq`.

Reply:
It does not